install.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+cd /home/pi
+
+# Update the base packages that come with the system. This is required
+# so that we can install git
+
+sudo apt update
+sudo apt upgrade -y
+sudo apt install git -y
+
+mkdir -p mozilla-iot
+cd mozilla-iot
+
+# Install and configure nvm
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash
+# The following 2 lines are installed into ~/.bashrc by the above,
+# but on the RPi, sourcing ~/.bashrc winds up being a no-op (when sourced
+# from a script), so we just run it here.
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && source "$NVM_DIR/nvm.sh"  # This loads nvm
+nvm install v7.10.1
+nvm use v7.10.1
+
+# Download, build, and install OpenZWave
+sudo apt install libusb-1.0-0-dev libudev-dev -y
+if [ ! -d "open-zwave" ]; then
+    git clone https://github.com/OpenZWave/open-zwave.git
+fi
+cd open-zwave
+make
+sudo make install
+sudo ldconfig /usr/local/lib
+
+# Download and install the required node modules
+cd ..
+if [ ! -d "gateway" ]; then
+    git clone https://github.com/moziot/gateway.git
+fi
+cd gateway
+npm install .
+
+# Setup systemctl so that the gateway will start at boot time
+sudo cp systemd/system/mozilla-iot-gateway.service /etc/systemd/system
+sudo systemctl enable mozilla-iot-gateway.service
+
+sudo cp gateway-iptables.sh /etc/init.d
```

Comment:
Heh - yeah - I have that change in my uncommitted changes.

I've also commented out creating the self-signed cert.